bundle-size/github-utils.js

```diff
@@ -218,21 +220,13 @@ class GitHubUtils {
   /**
    * Choose a random reviewer from list of potential approver teams.
    *
-   * @param {!Array<string>} potentialApproverTeams potential teams that can
-   *   approve this pull request.
    * @param {number} pullRequestId the pull request id.
+   * @param {!Array<string>} potentialReviewers list of GitHub usernames of all
+   *   users who are members of the teams that can approve the bundle-size
+   *   change of this pull request.
    * @return {string} the chosen reviewer username.
    */
-  async getRandomReviewer(potentialApproverTeams, pullRequestId) {
-    const teamIds = await Promise.all(
-      potentialApproverTeams.map(this.getTeamId_, this)
-    );
-    this.log(
-      `Reviewer for pull request ${pullRequestId} will be chosen from ` +
-        `[${potentialApproverTeams.join(', ')}] (team ids are ` +
-        `[${teamIds.join(', ')}])`
-    );
-    const potentialReviewers = await this.getTeamMembers_(teamIds);
+  async getRandomReviewer_(pullRequestId, potentialReviewers) {
```

Comment:
This may not be new, but it's the first time I'm noticing it. I don't think adding an argument to the function just to include it in a log is worthwhile, and it's a bit misleading (I saw it and immediately assumed it was making some GitHub API call, esp. since this method is `async` (also, should it be?)). This logging can happen in calling code

Reply:
Makes sense. This was probably a side effect of refactoring it as a method extraction